gulpfile.js

```diff
@@ -1,32 +1,77 @@
 var gulp = require('gulp');
 var uglify = require('gulp-uglify');
 var replace = require('gulp-replace');
-var minifyCss = require('gulp-minify-css');
+var stylus = require('gulp-stylus');
+var sourcemaps = require('gulp-sourcemaps');
+var browserify = require('browserify');
+var buffer = require('vinyl-buffer');
+var transform = require('vinyl-transform');
+var exorcist = require('exorcist');
```

Comment:
Oh, remove it. I'm not using it anymore.



—
Sent from Mailbox

On Tue, Feb 17, 2015 at 2:33 PM, Christian Mata <notifications@github.com>
wrote:

>> @@ -1,32 +1,77 @@
>>  var gulp = require('gulp');
>>  var uglify = require('gulp-uglify');
>>  var replace = require('gulp-replace');
>> -var minifyCss = require('gulp-minify-css');
>> +var stylus = require('gulp-stylus');
>> +var sourcemaps = require('gulp-sourcemaps');
>> +var browserify = require('browserify');
>> +var buffer = require('vinyl-buffer');
>> +var transform = require('vinyl-transform');
>> +var exorcist = require('exorcist');
> What is exorcist supposed to be used for? 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/DesignOpen/ResourceCollector/pull/13/files#r24856174